kolibri/content/utils/paths.py

```diff
@@ -1,13 +1,22 @@
 import os
+import re
 
 from django.conf import settings
+from django.core.urlresolvers import reverse
 
 try:
     from urlparse import urljoin
 except ImportError:
     from urllib.parse import urljoin
 
 
+# valid storage filenames consist of 32-char hex plus a file extension
+VALID_STORAGE_FILENAME = re.compile("[0-9a-f]{32}\.[0-9a-z]+")
```

Comment:
We would need to make a full list of extensions, as this function using this variable handles locating all types of files, not just the zipped ones. The main reason for `VALID_STORAGE_FILENAME` is safety validation, to ensure there's no funny business with `..` escaping out of the content directory, etc.